src/Halogen/DatePicker/Component/Date.purs

```diff
@@ -0,0 +1,182 @@
+module Halogen.Datapicker.Component.Date where
+
+import Prelude
+
+import Data.Bifunctor (bimap)
+import Data.Date (Date, Day, Month, Year)
+import Data.Either (Either(..))
+import Data.Either.Nested (Either2)
+import Data.Enum (class BoundedEnum, fromEnum, toEnum, upFromIncluding)
+import Data.Functor.Coproduct (Coproduct, coproduct, right, left)
+import Data.Functor.Coproduct.Nested (Coproduct2)
+import Data.Generic.Rep (class Generic)
+import Data.Generic.Rep.Show (genericShow)
+import Data.Array (sort)
+import Data.Maybe (Maybe(..), maybe)
+import Data.Monoid (mempty)
+import Data.Newtype (unwrap)
+import Data.NonEmpty (NonEmpty)
+import Data.Profunctor.Join (Join(..))
+import Data.Profunctor.Star (Star(..))
+import Data.Traversable (fold, for, sequence)
+import Halogen as H
+import Halogen.Component.ChildPath as CP
+import Halogen.Datapicker.Format.Date as F
+import Halogen.Datapicker.Internal.Choice as Choice
```

Comment:
Could you provide `Halogen.Datepicker.Internal` reexporting all package content?